changelogs/fragments/180-ec2_eni-stabilisation.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+- ec2_eni - Improve reliability of the module (https://github.com/ansible-collections/amazon.aws/pull/180).
```

Comment:
@tremble Nit, the waiters improve stability but can also make the module take longer to return. Can you expand on that in the changelog please?